pkgs/by-name/ru/runapp/package.nix

```diff
@@ -0,0 +1,50 @@
+{
+  fetchFromGitHub,
+  gcc15,
+  lib,
+  nix-update-script,
+  overrideCC,
+  pkg-config,
+  stdenv,
+  systemd,
+}:
+let
+  stdenv' = overrideCC stdenv gcc15;
```

Comment:
What is the reason to override the stdenv CC to gcc15?

Reply:
You are right, I just modified it :)